tox.ini

```diff
@@ -27,3 +26,10 @@ commands =
     ruff format src/ tests/ {posargs}
     mypy {posargs}
 
+[testenv:lint-github]
+description = Linting and formatting with ruff, typing with mypy
```

Comment:
You should add the licenser checker with --action=check-strict

Reply:
Like minds, already done ;)